app/pb_kits/playbook/pb_title/title.rb

```diff
@@ -13,7 +13,7 @@ class Title
                   values: [1, 2, 3, 4],
                   default: 3
       prop :tag, type: Playbook::Props::Enum,
-                 values: %w[h1 h2 h3 h4 h5 h6 p div span],
+                 values: %w[h1 h2 h3 h4 h5 h6 div],
```

Comment:
This could possibly break Nitro kits that are using the `tag` prop as `p` or `span`. See this for more: https://github.com/powerhome/playbook/pull/860/files#r442490239

Reply:
Hi! Thanks for the comment. I'm good with whatever you think should happen so if you think that I should put the tags back, I'll do it.